h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoost.java

```diff
@@ -412,7 +417,9 @@ private String createFeatureMapFile(String featureMap) {
       OutputStream os = null;
       try {
         File tmpModelDir = java.nio.file.Files.createTempDirectory("xgboost-model-" + _result.toString()).toFile();
```

Comment:
it seems to me we do not need to create this directory at all, the file-name should be quite unique by itself, or we can add the the _result the the file-name, rather than creating unnecessary directories